vllm/entrypoints/openai/serving_engine.py

```diff
@@ -100,6 +100,7 @@ def _create_logprobs(
                 token_logprob = step_top_logprobs[token_id].logprob
                 token = step_top_logprobs[token_id].decoded_token
                 logprobs.tokens.append(token)
+                token_logprob = max(token_logprob, -9999.0)
```

Comment:
Is there any comment you can leave to explain why this specific number?

Reply:
@mgoin  This number already been in used in [line 111](https://github.com/vllm-project/vllm/blob/dfc07c7a43375504494bbdfd48a386263071b214/vllm/entrypoints/openai/serving_engine.py#L111) for case like `-inf`. I took from there.